src/components/language-selector.js

```diff
@@ -16,6 +16,7 @@ function LanguageSelector({ slug }) {
   };
 
   const languageButtonGroup = languages.map((language) => (
+    <div className="banner__language-selector-item">
```

Comment:
Do you need to create a new `div`, or can you add a `class` attribute in the inner `Button`?